Library/Homebrew/shims/scm/git

```diff
@@ -92,7 +91,7 @@ case "$(lowercase "$SCM_FILE")" in
     ;;
 esac
 
-brew_version="$(quiet_safe_cd "$SCM_DIR/../../../../bin" && pwd -P)/$SCM_FILE"
+brew_version="$(quiet_safe_cd "$HOMEBREW_PREFIX/bin" && pwd -P)/$SCM_FILE"
```

Comment:
Is it possible to detect `HOMEBREW_PREFIX` in the same way `bin/brew` does here, though? There it finds it by looking at the directory of `bin/brew`, since it's symlinked into the prefix. That wouldn't work for this script, right?